env/test/test.properties

```diff
@@ -1 +1,6 @@
-OPERATOR_ENV=prod
\ No newline at end of file
+CHANNEL = ocp-4.4
+OPERATOR_VERSION = v1.0.1
+CSV = openshift-pipelines-operator.
+TKN_VERSION = 0.9.0
+OPERATOR_ENV = prod
+TEST_TYPE = pipelines
```

Comment:
instead of `TEST_TYPE` can we have `INSTALL_PIPELINE` with value `yes` or `no`?

Reply:
Make sense, will update it